dash/requirements.txt

```diff
@@ -0,0 +1,6 @@
+pandas
+matplotlib
+numpy
+plotly
+dash
+dotenv
```

Comment:
Do you think we should have one large requirements that sits in the overall directory? Given all of it is going to be put in the EC2.